domains/grawish.json

```diff
@@ -0,0 +1,15 @@
+{
+    "owner": {
+        "username": "grawish",
+        "email": "admin@grawish.com"
+    },
+
+    "target": {
+        "CNAME": {
+            "name": "grawish",
+            "value": ["grawish.com"]
```

Comment:
CNAME record values should be a single string with no arrays. This is not valid.